.eslintignore

```diff
@@ -1,4 +1,6 @@
 /.nyc_output/
+/eslint-local-rules/index.js
+/eslint-local-rules/dist/
```

Comment:
I tried putting these in a `.eslintignore` within the `eslint-local-rules/` dir, but it didn't work

Reply:
Fixed it